ingest/src/server_iot.rs

```diff
@@ -397,3 +409,7 @@ pub async fn grpc_server(settings: &Settings) -> Result<()> {
         .start()
         .await
 }
+
+fn pub_key_to_b58(pub_key: &[u8]) -> String {
```

Comment:
It looks like it would be useful to provide a `custom_tracing::record_b58/2`, or export `pub_key_to_b58` from custom_tracing.